### PR TITLE
modal and alerts css fixes

### PR DESCRIPTION
**Pull request by monika-Dianomic**
- [x] Fixed position of delete icon in readings graph modal.
- [x] Added header to all modals.
- [x] Changed dark alerts.

### CODE REVIEW

File: src/app/components/common/alert-dialog/alert-dialog.component.ts [hunk 1]
```diff
@@ -25,16 +25,16 @@ export class AlertDialogComponent implements OnInit, OnChanges {
 
   ngOnChanges() {
     if (this.childData) {
-      if (this.childData.key === 'restoreBackup') {
+      if (this.childData.key === 'Restore Backup') {
```
Comment (praveen-garg) [inline src/app/components/common/alert-dialog/alert-dialog.component.ts]: please don't change the keys

File: src/app/components/core/certificate/certificate-store/certificate-store.component.ts [hunk 2]
```diff
@@ -85,6 +85,7 @@ export class CertificateStoreComponent implements OnInit {
       message: message,
       key: action
     };
+    console.log('action', action);
```
Comment (praveen-garg) [inline src/app/components/core/certificate/certificate-store/certificate-store.component.ts]: remove this log

File: src/app/components/core/user-management/user-management.component.html [hunk 3]
```diff
@@ -51,7 +51,7 @@ <h6 class="is-6 has-text-weight-semibold">
                 </a>
               </td>
               <td *ngIf='user.userId != uid && user.userId != 1'>
-                <a (click)="openModal(user.userId, user.userName, 'deleteUser', 'Are you sure, You want to delete the user ')" class="button is-text">
+                <a (click)="openModal(user.userId, user.userName, 'deleteUser', 'Are you sure, You want to delete the ')" class="button is-text">
```
Comment (praveen-garg) [inline src/app/components/core/user-management/user-management.component.html]: don't delete the `user`, this is to alert 

> Are you sure, you want to delete the user *username*

File: src/app/components/core/user-management/user-management.component.html [hunk 4]
```diff
@@ -51,14 +51,14 @@ <h6 class="is-6 has-text-weight-semibold">
                 </a>
               </td>
               <td *ngIf='user.userId != uid && user.userId != 1'>
-                <a (click)="openModal(user.userId, user.userName, 'deleteUser', 'Are you sure, You want to delete the user ')" class="button is-text">
+                <a (click)="openModal(user.userId, user.userName, 'deleteUser', 'Are you sure, you want to delete the user')" class="button is-text">
```
[the remark this answers is not in the document]
Comment (monika-Dianomic) [reply src/app/components/core/user-management/user-management.component.html]: It is not required here.

File: src/app/components/core/user-management/user-management.component.html [hunk 5]
```diff
@@ -40,25 +44,25 @@ <h6 class="is-6 has-text-weight-semibold">
...
-                <a (click)="openUpdateUserModal(user, 'password')" class="button is-text">
+              <td>
+                <a *ngIf='user.userId != uid && user.userId != 1' (click)="openUpdateUserModal(user, 'password')" class="button is-text">
                   reset password
                 </a>
               </td>
-              <td *ngIf='user.userId != uid && user.userId != 1'>
-                <a (click)="openModal(user.userId, user.userName, 'deleteUser', 'Are you sure, You want to delete the user ')" class="button is-text">
+              <td>
+                <a *ngIf='user.userId != uid && user.userId != 1' (click)="openModal(user.userId, user.userName, 'deleteUser', 'Are you sure, you want to delete the user')" class="button is-text">
```
Comment (praveen-garg) [inline src/app/components/core/user-management/user-management.component.html]: should we use ng-container here?

File: src/app/components/core/user-management/user-management.component.html [hunk 6]
```diff
@@ -40,25 +44,25 @@ <h6 class="is-6 has-text-weight-semibold">
...
                   <i class="fa fa-minus-circle" aria-hidden="true"></i>
                 </a>
               </td>
               <td>
                 <span *ngIf='user.userId == 1' class="tag is-rounded">super admin</span>
                 <span *ngIf='user.userId == uid' class="tag is-rounded">active</span>
```
Comment (praveen-garg) [inline src/app/components/core/user-management/user-management.component.html]: We need to move these into nearest td for admin user row

File: src/app/components/core/configuration-manager/config-children/config-children.component.css [hunk 7]
```diff
@@ -1,6 +1,8 @@
-label{
-  width: 12vw !important;
-  margin-right: 2vw;
+@media screen and (min-width: 769px) {
```
Comment (praveen-garg) [inline src/app/components/core/configuration-manager/config-children/config-children.component.css]: We also need the same for other config views

File: src/app/components/common/restart-modal/restart-modal.component.html [hunk 8]
```diff
@@ -1,12 +1,15 @@
...
-          <button class="button is-small" (click)="toggleModal(false)">Cancel</button>
-      </footer>
-  </div>
-</div>
+    <div class="modal-background"></div>
+    <div class="modal-card">
+        <header class="modal-card-head">
+            <p class="modal-card-title is-size-6">Restart FogLAMP</p>
+        </header>
+        <section class="modal-card-body">
```
Comment (praveen-garg) [inline src/app/components/common/restart-modal/restart-modal.component.html]: padding: 2rem 1.5rem;

File: src/app/components/common/shut-down/shutdown-modal.component.html [hunk 9]
```diff
@@ -1,12 +1,15 @@
 <div id="shutdownModal-box" class="modal">
     <div class="modal-background"></div>
     <div class="modal-card">
+        <header class="modal-card-head">
+            <p class="modal-card-title is-size-6">Shut Down FogLAMP</p>
+        </header>
         <section class="modal-card-body">
```
Comment (praveen-garg) [inline src/app/components/common/shut-down/shutdown-modal.component.html]: padding: 2rem 1.5rem;

File: src/app/services/alert.service.ts [hunk 10]
```diff
@@ -2,6 +2,8 @@ import { Injectable } from '@angular/core';
 import { NavigationStart, Router } from '@angular/router';
 import { Observable, Subject } from 'rxjs';
 
+export const DARK_ALERTS = 0;
```
Comment (praveen-garg) [inline src/app/services/alert.service.ts]: set it to 1, to keep the behavior as of develop

**Pull request merged**